library/private/options_tools.h.in

```diff
@@ -103,7 +135,7 @@ double parse(const std::string& str)
   // TODO implement more parsing possibilities ?
   try
   {
-    return std::stod(str);
+    return parse_double(str);
```

Comment:
supports NaN and Inf ?

Reply:
should be possible

Reply:
possible but current use-cases would be better implemented with proper optional value support

Reply:
indeed